bridges/otelzap/core_test.go

```diff
@@ -53,3 +71,26 @@ func TestNewCoreConfiguration(t *testing.T) {
 		assert.Equal(t, want, got)
```

Comment:
Add tests for `Write` and `Check`?

Reply:
Personally, I am not sure if it is needed given we test these methods (both of them) via `TestCore`.